src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+
+.product-card__name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__view {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: center;
+  align-content: flex-end;
+  line-height: 14px;
+}
+
+.product-card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  align-items: center;
+}
+
+.product-card__text {
+  color: #616070;
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.product-card__sum {
+  font-weight: 700;
+  line-height: 18px;
+  font-size: 16px;
+}
+
+.product-card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 164px;
```

Comment:
don`t hardcore with such exact values when you can go easier - use just 100%)

Reply:
ok, i will fix this mistake.